packages/cli/src/commands/down.ts

```diff
@@ -28,11 +28,11 @@ export default class Down extends DriverCommand<typeof Down> {
     const log = this.logger
     const { flags } = await this.parse(Down)
     const driver = await this.driver()
-    const userModel = await this.ensureUserModel()
-    const projectName = userModel.name
-    log.debug(`project: ${projectName}`)
-    const envId = flags.id ?? await findAmbientEnvId(projectName)
-
+    const [envId, userModel] = flags.id ? [flags.id, { name: flags.id }] : await (async () => {
```

Comment:
`envId` is incorrect here - missing branch suffix.
what is this fixing?

Reply:
The case of using down --id without docker/compose. (#114)

Reply:
if the project name is only needed for the `debug.log` call, I suggest removing it to simplify:
```suggestion
    const envId = flags.id ?? await findAmbientEnvId((await this.ensureUserModel()).name)
```

Reply:
That was the variable before.
```const envId = flags.id ?? await findAmbientEnvId(projectName)```